autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -87,6 +89,10 @@ def get_preset_models_classification(path, problem_type, objective_func, stoppin
             TabularNeuralNetModel(path=path, name='NeuralNetClassifier', problem_type=problem_type,
                                   objective_func=objective_func, stopping_metric=stopping_metric, hyperparameters=nn_options.copy()),
         )
+    if lr_options is not None:
```

Comment:
Regression is still not added, it can't be specified as a model in a regression problem. It has to be added in get_preset_models_regression()